py3xui/utils/env.py

```diff
@@ -72,4 +74,18 @@ def xui_password() -> str:
     return parse_env(
         keys=["XUI_PASSWORD"],
         postprocess_fn=lambda x: x,
+    )  # type: ignore[return-value]
+
+
+def xui_token() -> str | None:
```

Comment:
Should we add the `required` argument to all env functions?
E.g. user specifically wants to raise an error when the parameters are not found? And for required parameters set default to True, for optional to False.
Maybe we should rename the `required` argument to something more specific? E.g. `raise_if_not_found`?